### PR TITLE
Add emojis to pipeline/pipelinerun/task/taskrun describe

### PR DESCRIPTION
**Pull request by chmouel**
Not showing when we don't want colours (or not a real terminal).

The choice of emojis are fairly abritrary, I am open for suggestions :D

Some screenshots : 

## Pipeline

image

## Task

image

## TaskRun

image

### CODE REVIEW

File: pkg/cmd/pipeline/describe.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ import (
 const describeTemplate = `{{decorate "bold" "Name"}}:	{{ .PipelineName }}
 {{decorate "bold" "Namespace"}}:	{{ .Pipeline.Namespace }}
 
-{{decorate "underline bold" "Resources\n"}}
+{{decorate "resources" ""}}{{decorate "underline bold" "Resources\n"}}
```
Comment (chmouel) [inline pkg/cmd/pipeline/describe.go]: tried to do stuff like this : 

```
{{ decorate "inputresources" "" | decorate "underline bold" "Input Resources\n"}}
``

but I am getting the error message : 

```
Error: template: Describe Task:4:33: executing "Describe Task" at <decorate>: wrong number of args for decorate: want 2 got 3
```
🤔

**Comment by chmouel** [comment]
/retest

failure was : 
```
I0122 09:51:44.309]  detail: u"All cluster resources were brought up, but: only 2 nodes out of 3 have registered; this is likely due to Nodes failing to start correctly; try re-creating the cluster or contact support if that doesn't work."
```

File: pkg/formatted/color.go [hunk 2]
```diff
@@ -41,10 +41,29 @@ func DecorateAttr(attrString, message string) string {
...
+		return fmt.Sprintf("🗒  ")
+	case "pipelineruns":
+		return fmt.Sprintf("⛩  ")
+	case "status":
+		return fmt.Sprintf("🌡️  ")
+	case "inputresources":
+		return fmt.Sprintf("📨 ")
+	case "outputresources":
+		return fmt.Sprintf("📡 ")
+	case "steps":
```
Comment (danielhelfand) [inline pkg/formatted/color.go]: 👣 👍

Comment (chmouel) [reply pkg/formatted/color.go]: I didnt take the black feet version due of not showing very well on dark terminal... i guess those one works on dark and light background terminal....

Comment (danielhelfand) [reply pkg/formatted/color.go]: Oh yeah, no worries! More approving and wanted to mix it up.

**Pull request merged**